pillar.example

```diff
@@ -96,7 +96,7 @@ users:
     gitconfig:
       user.name: B User
       user.email: buser@example.com
-      url."https://".insteadOf: "git://"
+      "url.https://.insteadOf": "git://"
```

Comment:
In [this link](https://coderwall.com/p/sitezg/force-git-to-clone-with-https-instead-of-git-urls) and [this other link](http://stackoverflow.com/a/5010453) I see that the previous syntax (although weird) is the correct one, as quotes enclose the replacement URI to use instead of the rvalue given.

Have you tried this? I'm not sure it will work for all the possible URIs that can be specified?